charts/ccd-case-disposer/values.preview.template.yaml

```diff
@@ -117,7 +117,7 @@ ccd:
       environment:
         DATA_STORE_DB_HOST: ${SERVICE_NAME}-postgresql
         DATA_STORE_IDAM_KEY: ${CCD_DATA_STORE_S2S_SECRET}
-        DATA_STORE_S2S_AUTHORISED_SERVICES: ccd_gw,ccd_gateway,fpl_case_service,ccd_data,ccd_ps
+        DEFINITION_STORE_S2S_AUTHORISED_SERVICES: ccd_gw,ccd_gateway,fpl_case_service,ccd_data,ccd_ps
         IDAM_USER_URL: https://idam-web-public.aat.platform.hmcts.net
```

Comment:
should this be in the DEFINITION_STORE section?

Reply:
And presumably needs the `ccd_case_disposer` to be an authorised service?